### ISSUE TITLE
issue installing v1.5: yarn dev fails (because yarnpkg @calcom/app-store is not available?)

### ISSUE THREAD

**Issue by wivaku** [calcom/cal.com#2531]
Following [instructions to install](https://docs.cal.com/self-hosting/install) v1.5 locally.

- using `yarn set version stable`, which sets it to 3.2.0
- using node v17.9.0
- for step 4 of the setup instructions ("install packages with yarn") I used `yarn install` (just `yarn` did not work)
- step 3 of the "manual setup" instructions (DB deploy) works as expected: `yarn workspace @calcom/prisma db-deploy`
- step 4 of the "manual setup" instructions (run) fails:
  - `yarn dev` fails with error message: `Internal Error: @calcom/web@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile`
  - running `yarn install` does the same thing as before, and does not change the outcome
  - I tried instead: `yarn workspace @calcom/web install`
  - that one ends with an error message: `@calcom/app-store@npm:*: The remote server failed to provide the requested resource - https://registry.yarnpkg.com/@calcom%2fapp-store not found`
  - indeed, `https://yarnpkg.com/package/@calcom/app-store` does not exist

```
╰─❯ yarn workspace @calcom/web install
➤ YN0000: ┌ Resolution step
➤ YN0061: │ @jitsu/sdk-js@npm:2.3.0 is deprecated: versions prior to 2.5.4 do not explicitly specify path in user ID cookie which may break tracking
➤ YN0061: │ @types/stripe@npm:8.0.417 is deprecated: This is a stub types definition. stripe provides its own type definitions, so you do not need this installed.
➤ YN0061: │ webextension-polyfill-ts@npm:0.25.0 is deprecated: This project has moved to @types/webextension-polyfill
➤ YN0061: │ webextension-polyfill-ts@npm:0.22.0 is deprecated: This project has moved to @types/webextension-polyfill
➤ YN0061: │ uuid@npm:3.3.2 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ sane@npm:4.1.0 is deprecated: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
➤ YN0061: │ mkdirp-promise@npm:5.0.1 is deprecated: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
➤ YN0061: │ @types/socket.io-parser@npm:3.0.0 is deprecated: This is a stub types definition. socket.io-parser provides its own type definitions, so you do not need this installed.
➤ YN0032: │ evp_bytestokey@npm:1.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ source-map-resolve@npm:0.5.3 is deprecated: See https://github.com/lydell/source-map-resolve#deprecated
➤ YN0061: │ multicodec@npm:0.5.7 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: │ cids@npm:0.7.5 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: │ multicodec@npm:1.0.4 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: │ resolve-url@npm:0.2.1 is deprecated: https://github.com/lydell/resolve-url#deprecated
➤ YN0061: │ urix@npm:0.1.0 is deprecated: Please see https://github.com/lydell/urix#deprecated
➤ YN0032: │ keccak@npm:3.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ source-map-url@npm:0.4.1 is deprecated: See https://github.com/lydell/source-map-url#deprecated
➤ YN0032: │ secp256k1@npm:4.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ bufferutil@npm:4.0.6: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ utf-8-validate@npm:5.0.8: Implicit dependencies on node-gyp are discouraged
➤ YN0032: │ node-addon-api@npm:2.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ request@npm:2.88.2 is deprecated: request has been deprecated, see https://github.com/request/request/issues/3142
➤ YN0061: │ uuid@npm:3.4.0 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: │ har-validator@npm:5.1.5 is deprecated: this library is no longer supported
➤ YN0061: │ multibase@npm:0.6.1 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: │ multibase@npm:0.7.0 is deprecated: This module has been superseded by the multiformats module
➤ YN0035: │ @calcom/app-store@npm:*: The remote server failed to provide the requested resource
➤ YN0035: │   Response Code: 404 (Not Found)
➤ YN0035: │   Request Method: GET
➤ YN0035: │   Request URL: https://registry.yarnpkg.com/@calcom%2fapp-store
➤ YN0000: └ Completed in 16s 186ms
➤ YN0000: Failed with errors in 16s 193ms
```

**Comment by agustif** [comment]
I think the problem here is you need to use turborepo to install everything from the monorepo.

When you run install from web workspace directly, probably the rest of the scoped packages in the monorepo (all the @calcom/*) are not available to your web package if you don't install the rest of the monorepo?

I'd try running yarn install in the root monorepo, and then running any of the workspaces by itself should be fine.

LMK if that works or how else can we help, sorry for the trouble!

**Comment by wivaku** [comment]
That was quick, thanks.
With "running yarn install in the root monorepo" do you mean this?
`cd node_modules/@calcom && yarn install`

That works, but the `yarn dev` afterwards (in root folder) fails same as before.

FYI similar when using node v14.19.1
- step 4 ("install packages with yarn"): `yarn` works (no `yarn install` needed)
- step 4 ("manual setup"):`yarn dev` fails same as before

**Comment by wivaku** [comment]
I may be doing this completely wrong, but I tried: `yarn turbo run build`

Which complains about: `@calcom/embed-core@workspace` missing from lockfile, I ran`yarn workspace @calcom/embed-core install` and tried `yarn turbo run build` again.

Then it complained about `@calcom/swagger@workspace`, I ran `yarn workspace @calcom/swagger install` and tried `yarn turbo run build` again.

Then it complains about `@calcom/docs@workspace`, I ran `yarn workspace @calcom/docs install`
but this one fails: 

```
╰─❯ yarn workspace @calcom/docs install
➤ YN0000: ┌ Resolution step
➤ YN0035: │ @calcom/config@npm:*: The remote server failed to provide the requested resource
➤ YN0035: │   Response Code: 404 (Not Found)
➤ YN0035: │   Request Method: GET
➤ YN0035: │   Request URL: https://registry.yarnpkg.com/@calcom%2fconfig
➤ YN0000: └ Completed in 4s 512ms
➤ YN0000: Failed with errors in 4s 520ms
```

Tried `yarn workspace @calcom/config install` and then `yarn workspace @calcom/docs install` but same result.
